ivy/functional/frontends/tensorflow/functions.py

```diff
@@ -27,16 +27,22 @@ def fill(dims, value, name="full"):
 fill.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
 
 
-
 def multiply(x, y, name=None):
     return ivy.multiply(x, y)
 
 
 multiply.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
 
+
 def subtract(x, y, name=None):
     return ivy.subtract(x, y)
 
 
 subtract.unsupported_dtypes = {"torch": ("float16", "bfloat16")}
 
+
+def average(x, y, name=None):
+    return ivy.average(x, y)
```

Comment:
`ivy.average` doesn't exist, you will need to implement this yourself as a composition function, or you can try to use `ivy.mean`